InvenTree/InvenTree/helpers.py

```diff
@@ -145,12 +145,20 @@ def download_image_from_url(remote_url, timeout=2.5):
     # Calculate maximum allowable image size (in bytes)
     max_size = int(InvenTreeSetting.get_setting('INVENTREE_DOWNLOAD_IMAGE_MAX_SIZE')) * 1024 * 1024
 
+    # Add user specified user-agent to request (if specified)
+    user_agent = str(InvenTreeSetting.get_setting('INVENTREE_DOWNLOAD_FROM_URL_USER_AGENT'))
```

Comment:
I think you do not want to cast this to a string. If `get_setting` returns None, you'll actually get "None"